docs/framework/migration-guide/application-compatibility.md

```diff
@@ -15,128 +15,81 @@ helpviewer_keywords:
-- [Application Compatibility in 4.7](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-7.md)
-  
-## Migrating from the .NET Framework 4.6  
- If you are migrating from the .NET Framework 4.6 to a later version, review the following topics for application compatibility issues that may affect your app:  
-  
--   [Application Compatibility in 4.6.1](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-6-1.md)  
-  
--   [Application Compatibility in 4.6.2](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-6-2.md)  
-
-- [Application Compatibility in 4.7](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-7.md)
-  
-## Migrating from the .NET Framework 4.6.1  
- If you are migrating from the .NET Framework 4.6.1, review the following topics for application compatibility issues that may affect your app:
- 
-- [Application Compatibility in 4.6.2](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-6-2.md)
-
-- [Application Compatibility in 4.7](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-7.md)  
-
-## Migrating from the .NET Framework 4.6.2
-
-If you are migrating from the .NET Framework 4.6.2, review [Application Compatibility in 4.7](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-7.md) for application compatibility issues that may affect your app.  
-
-## See Also  
- [Versions and Dependencies](../../../docs/framework/migration-guide/versions-and-dependencies.md)   
- [What's New](../../../docs/framework/whats-new/index.md)   
- [What's Obsolete in the Class Library](../../../docs/framework/whats-new/whats-obsolete.md)
\ No newline at end of file
+
+## Introduction
+Compatibility is a very important goal of each .NET release. This is
+accomplished by ensuring each version is additive so previous versions will
+still work; however, when a change must be made to previous functionality (for
+performance, security, or a bug) this can cause compatibility problems in
+already written code. A distinction to understand with .NET compatibility is
+runtime vs retargeting issues.
+
+When developing an app the .NET Framework version that the app is targeting is
+specified either by specifying a target Framework in Visual Studio, specifying
+the target Framework in a project file, or applying a TargetFrameworkAttribute
+to the source code.
+
+When running on a newer version than what was targeted, the .NET Framework will
+use quirked behavior to mimic the older targeted version. In other words, the
+app will run on the newer version of the Framework, but act as if it's running
+on the older version. Many of the breaking changes between versions of the .NET
+Framework are mitigated through this quirking model.
+
+Runtime issues are those that arise when a new runtime is placed on a machine
```

Comment:
I think the paragraph describes runtime issues rather than retargeting issues.  Runtime issues are differences that are seen when a user installs a newer version of .NET Framework on their machine and then runs their application (which is targeting an older version).  Retargeting changes are differences during runtime when users recompile their code against a newer version of the .NET Framework.

From [Runtime Changes in the .NET Framework 4.5.1](https://msdn.microsoft.com/en-us/library/dn458360(v=vs.110).aspx)
> ... runtime changes may affect existing apps that target the .NET Framework 4 or 4.5 but run on the 4.51 runtime.

From [Retargeting Changes in the .NET Framework 4.5.1](https://msdn.microsoft.com/en-us/library/dn458356(v=vs.110).aspx)
> ...retargeting changes may affect apps that are recompiled to target the .NET Framework 4.5.1. 


@twsouthwick :